Generals/Code/GameEngine/Source/GameClient/MessageStream/WindowXlat.cpp

```diff
@@ -294,8 +295,9 @@ GameMessageDisposition WindowTranslator::translateGameMessage(const GameMessage
 
 			// process event through window system
 			if( TheWindowManager )
+			{
```

Comment:
This edit look different to Generals MD. Please make a pass and eliminate all differences in edits between the games. Both games must be touched identically.